src/flood_forecaster/utils/database_helper.py

```diff
@@ -0,0 +1,127 @@
+from sqlalchemy.engine import URL
+from sqlalchemy.exc import SQLAlchemyError
+
+
+class DatabaseConnection:
+    def __init__(self, config_file_path: str) -> None:
+        """
+        Initialize the database connection using parameters from a config file.
+
+        :param config_file_path: Path to the configuration file
+        """
+        self.config = self._load_config(config_file_path)
+        self.dbname = self.config.get("database", "dbname")
+        self.user = self.config.get("database", "user")
+        self.host = self.config.get("database", "host")
+        self.port = self.config.get("database", "port")
+        self.password = os.environ.get("POSTGRES_PASSWORD")
+
+        try:
+            url = URL.create(
+                drivername="postgresql",
+                username=self.user,
+                password=self.password,
+                host=self.host,
+                port=self.port,
+                database=self.dbname
+            )
+            self.engine = create_engine(url)
+            print(f"Connected to database '{self.dbname}'")
+        except SQLAlchemyError as e:
+            print(f"Failed to connect to database: {str(e)}")
+            raise
+
+    @staticmethod
+    def _load_config(config_file_path: str) -> configparser.ConfigParser:
+        """
+        Load configuration from the given file path.
+
+        :param config_file_path: Path to the configuration file
+        :return: ConfigParser object
+        """
+        if not os.path.exists(config_file_path):
+            raise FileNotFoundError(f"Config file '{config_file_path}' not found.")
+        
+        config = configparser.ConfigParser()
+        config.read(config_file_path)
+        return config
+
+    def create_schema(self, schema_name: str) -> None:
+        """
+        Create a schema in the database.
+        
+        :param schema_name: Name of the schema to create
+        """
+        try:
+            with self.engine.connect() as connection:
+                connection.execute(text(f"CREATE SCHEMA IF NOT EXISTS {schema_name};"))
+                print(f"Schema '{schema_name}' created (or already exists).")
+        except SQLAlchemyError as e:
+            print(f"Error creating schema '{schema_name}': {str(e)}")
+
+    def create_tables(self, schema_name: str, base) -> None:
+        """
+        Create tables in the specified schema using ORM models.
+
+        :param schema_name: Name of the schema
+        :param base: SQLAlchemy Base containing ORM models
+        """
+        try:
+            with self.engine.begin() as connection:
+                connection.execute(text(f"SET search_path TO {schema_name};"))
+                base.metadata.create_all(bind=connection)
+                print(f"Tables created in schema '{schema_name}'.")
+        except SQLAlchemyError as e:
+            print(f"Error creating tables in schema '{schema_name}': {str(e)}")
+
+    def list_db_schemas(self) -> list:
```

Comment:
I see these last two methods as useful scripts like the ones in `schema_report/update.py`. I don't think they'll be called during the normal workflow of the application and that's why I'd prefer to put them separately, like in a `database_scripts` folder or something like that.

Reply:
Actually I call them from the schema_report.py, like the create_schemas/tables are called from schema_update.py script; and I see those scripts as part of the command-line tool, like:
`forecaster-client [database [--report] [--update [....]] [--upload [-f <file_name>]] ... 
                                [train-model ...]
                                ....` 
                                
So I just pushed those 2 scripts down to  the base flood_forecaster - so they could added to a client module later.